server/db/models/index.js

```diff
@@ -1,11 +1,16 @@
 const User = require('./user')
+const Product = require('./product.js')
+const Category = require('./category.js')
+const db = require('../db')
 
 /**
  * If we had any associations to make, this would be a great place to put them!
  * ex. if we had another model called BlogPost, we might say:
  *
  *    BlogPost.belongsTo(User)
  */
+Product.belongsToMany(Category, {through: 'Product-Category'})
+Category.belongsToMany(Product, {through: 'Product-Category'})
```

Comment:
Avoid capitalization and hyphens in SQL table names; there is actually an option to control this in Sequelize, as the default I think uses camelCase, but if you're setting the table name manually it ought to be `product_category`.